flow/core/params.py

```diff
@@ -19,6 +19,14 @@
 
 LC_MODES = {"aggressive": 0, "no_lat_collide": 512, "strategic": 1621}
 
+SUPPORTED_BASELINES = ['actuated', 'static']
```

Comment:
Probably shouldn't be titled baseline, it's not clear what that means outside of an RL context. As in, a lot of people wouldn't view these as a "baseline". Don't really have a good idea for a better title yet though

Reply:
I thought baseline would be a fairly understandable word. Open to other suggestions